src/main/java/com/training/utilities/RestEngine.java

```diff
@@ -26,4 +26,15 @@ public Response getResponsePost(String base_url, Map headerMap, String jsonBody)
                 .extract()
                 .response();
     }
+
+    public Response getResponsePutLoyality(String base_url, Map headerMap, String jsonBody) {
```

Comment:
Will this only be applicable for Loyalty?
If not, then change the method name to getResponsePut()